qt/languagedialog.py

```diff
@@ -12,6 +12,7 @@
 import tools
 import qttools
 import logger
+import languages
```

Comment:
I think this pgk needs to be added to the (optional) dev dependencies in our documentation:

https://github.com/bit-team/backintime/blob/dev/CONTRIBUTING.md#build--install